python/lsst/ctrl/mpexec/mpGraphExecutor.py

```diff
@@ -236,18 +246,29 @@ class MPGraphExecutor(QuantumGraphExecutor):
     quantumExecutor : `QuantumExecutor`
         Executor for single quantum. For multiprocess-style execution when
         ``numProc`` is greater than one this instance must support pickle.
+    startMethod : `str`, optional
+        Start method from `multiprocessing` module, `None` selects the best
+        one for current platform.
     failFast : `bool`, optional
         If set to ``True`` then stop processing on first error from any task.
     executionGraphFixup : `ExecutionGraphFixup`, optional
         Instance used for modification of execution graph.
     """
-    def __init__(self, numProc, timeout, quantumExecutor, *, failFast=False, executionGraphFixup=None):
+    def __init__(self, numProc, timeout, quantumExecutor, *,
+                 startMethod=None, failFast=False, executionGraphFixup=None):
         self.numProc = numProc
         self.timeout = timeout
         self.quantumExecutor = quantumExecutor
         self.failFast = failFast
         self.executionGraphFixup = executionGraphFixup
 
+        # By default use spawn for MacOS and fork for Linux
+        if startMethod is None:
+            methods = dict(linux="fork", darwin="spawn")
+            startMethod = methods.get(sys.platform)
```

Comment:
Maybe add a default value for the `get` of "spawn" in case you have an unexpected `sys.platform` value?

Reply:
Unexpected platform will get `None` which will use default `multiprocessing` method. I think that should work better because `spawn` may not work for that unknown platform.

Reply:
I'll expand the comment above to mention that.